server/api.go

```diff
@@ -530,12 +533,12 @@ func (a *API) oauthRedirectHandler(w http.ResponseWriter, r *http.Request) {
 	}
 
 	w.Header().Add("Content-Type", "text/html")
-	connectionMessage := "Your account has been connected"
 	if mmUser.Id == a.p.GetBotUserID() {
-		connectionMessage = "The bot account has been connected"
+		_, _ = w.Write([]byte("<html><body><h1>The bot account has been connected</h1><p>You can close this window.</p></body></html>"))
+		return
 	}
 
-	_, _ = w.Write([]byte(fmt.Sprintf("<html><body><h1>%s</h1><p>You can close this window.</p></body></html>", connectionMessage)))
+	http.Redirect(w, r, a.p.GetURL()+"/static/info-page/index.html", http.StatusFound)
```

Comment:
Before merging, we should comment this out and keep the original code. Mainly because we don't want to show this until we have all the pieces in place. What do you think?